scss/_settings_colors.scss

```diff
@@ -109,6 +109,8 @@ $colors--light-theme--text-inactive: rgba($color-x-dark, $inactive-text-opacity-
 $colors--light-theme--link-default: $color-link !default;
 $colors--light-theme--link-visited: $color-link-visited !default;
 
+$colors--light-theme--code-background-default: $color-code-background !default;
```

Comment:
I don't think we will need different code backgrounds, so I would treat it as a variant of background colour. 

So following out naming conventions it would be `$colors--light-theme--background-code` instead.

Please update all the names to follow this convention.

Reply:
where did this code background come from?